ush/python/pygfs/task/stage_ic.py

```diff
@@ -46,6 +48,24 @@ def execute_stage(self, stage_dict: Dict[str, Any]) -> None:
         -------
         None
         """
+        YYYYMMDDHH = to_YMDH(stage_dict.current_cycle)
+        YYYY = YYYYMMDDHH[0:4]
+        MM = YYYYMMDDHH[4:6]
+
+        if stage_dict.DO_DOWNLOAD_ICS is True:
+            # Download ICs from HPSS to ICSDIR
+            self.tar_cmd = "htar"
+            self.hsi = Hsi()
+            self.htar = Htar()
+            self.xvf = self.htar.xvf
+            self.xvf(stage_dict.HPSSICARCH + "/" + YYYYMMDDHH + ".tar")
+
+        if stage_dict.DO_REPAIR_REPLAY is True:
+            # Download f03 replay analysis
+            aws_cmd = "aws s3 cp --no-sign-request"
+            aws_url = "s3://noaa-ufs-gefsv13replay-pds/"
+            subprocess.run(aws_cmd + " " + aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSPRS.GrbF03 ./", shell=True)
+            subprocess.run(aws_cmd + " " + aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSFLX.GrbF03 ./", shell=True)
```

Comment:
```suggestion
            aws_cmd = which("aws")
            aws_cmd.add_default_arg("s3")
            aws_cmd.add_default_arg("cp")
            aws_cmd.add_default_arg("--no-sign-request")
            
            aws_url = "s3://noaa-ufs-gefsv13replay-pds/"
            aws_cmd([aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSPRS.GrbF03", "./"])
            aws_cmd([aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSFLX.GrbF03", "./"])
```

Reply:
@DavidHuber-NOAA When I try your suggestion, I get the following error (`/lfs/h2/emc/ptmp/eric.sinsky/GEFS/COMROOT/customexp/gw_reforecast_update/logs/1994010400/gefs_stage_ic.log.9`):
```
  File "/lfs/h2/emc/ens/noscrub/eric.sinsky/GEFS/gw_reforecast_update/ush/python/pygfs/task/stage_ic.py", line 70, in execute_stage
    aws_cmd([aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSPRS.GrbF03", "./"])
  File "/lfs/h2/emc/ens/noscrub/eric.sinsky/GEFS/gw_reforecast_update/ush/python/wxflow/executable.py", line 199, in __call__
    escaped_cmd = ["'%s'" % arg.replace("'", "'\"'\"'") for arg in cmd]
  File "/lfs/h2/emc/ens/noscrub/eric.sinsky/GEFS/gw_reforecast_update/ush/python/wxflow/executable.py", line 199, in <listcomp>
    escaped_cmd = ["'%s'" % arg.replace("'", "'\"'\"'") for arg in cmd]
AttributeError: 'list' object has no attribute 'replace'
```

I then removed the list object and changed line 70 to: 
`aws_cmd(aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSPRS.GrbF03 ./")`
However, when I do this, I get the following error (/lfs/h2/emc/ptmp/eric.sinsky/GEFS/COMROOT/customexp/gw_reforecast_update/logs/1994010400/gefs_stage_ic.log):
`aws: error: the following arguments are required: paths
`

Reply:
Whoops, that should not have been a list, but a set of strings sent in as arguments.  Please try this instead:

```suggestion
            aws_cmd = which("aws")
            aws_cmd.add_default_arg("s3")
            aws_cmd.add_default_arg("cp")
            aws_cmd.add_default_arg("--no-sign-request")
            
            aws_url = "s3://noaa-ufs-gefsv13replay-pds/"
            aws_cmd(aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSPRS.GrbF03", "./")
            aws_cmd(aws_url + YYYY + "/" + MM + "/" + YYYYMMDDHH + "/GFSFLX.GrbF03", "./")
```

Reply:
Thank you, @DavidHuber-NOAA. Comma-separating the set of strings fixed the issue.